### PR TITLE
New hmm download

### PR DESCRIPTION
**Pull request by LeeBergstrand**
Download and use the TIGRFAM HMM for contig rotation. I vaguely remember that this rule would only run if the end repair was running (my bypass code discussed in https://github.com/jmtsuji/rotary/issues/52). Could you integrate and test this while you are working on end repair?

### CODE REVIEW

File: rotary/rules/rotary.smk
Comment (LeeBergstrand) [reply rotary/rules/rotary.smk:760]: @jmtsuji We can probably do a clean-up pass over the code once we get things working and the testing suite in place.

File: rotary/rules/rotary.smk
Comment (LeeBergstrand) [reply rotary/rules/rotary.smk:87]: @jmtsuji So it's a double-edged sword. On the one hand, it would prevent specific attacks where an attacker could doppelganger as the server and send out malware as the hmm. On the other hand, it is more convenient if the server fails to get a new certificate. TI added it during debugging because they didn't bother to get a certificate for the legacy server. After all, it was temporary.

Comment (LeeBergstrand) [reply rotary/rules/rotary.smk:87]: My recommendation would be to leave it out unless we run into the issue again constantly in the future.

File: rotary/rules/rotary.smk
Comment (LeeBergstrand) [reply rotary/rules/rotary.smk:15]: Good eye! Removing.

File: rotary/rules/rotary.smk
Comment (LeeBergstrand) [reply rotary/rules/rotary.smk:87]: See: https://chat.openai.com/share/930460eb-2687-4e57-b683-e06ec3074eee for more info.

**Comment by LeeBergstrand** [comment]
@jmtsuji I'll let you merge this once you have reviewed my comments.

File: rotary/rules/rotary.smk
Comment (jmtsuji) [reply rotary/rules/rotary.smk:87]: OK, let's leave it out unless we have issues with this again in the future. I'll make a commit to this branch with the change.

**Review by jmtsuji — APPROVED** [review]

**Pull request merged**
